cmd/enduro-a3m-worker/main.go

```diff
@@ -212,6 +213,9 @@ func main() {
 			EnableSessionWorker:                true,
 			MaxConcurrentSessionExecutionSize:  cfg.A3m.Capacity,
 			MaxConcurrentActivityExecutionSize: 1,
+			Interceptors: []temporalsdk_interceptor.WorkerInterceptor{
+				temporal_tools.NewLoggerInterceptor(logger.WithName("a3m-worker")),
```

Comment:
The logger was already named `enduro-a3m-worker` (using `appName`) so I believe that the name of this child logger would be `enduro-a3m.worker.a3m-worker`, is that what you want?

Reply:
Good point, maybe not appending anything works better in this case.